Source/dom/layers/Layer.js

```diff
@@ -110,6 +110,14 @@ export class Layer extends WrappedObject {
     })
   }
 
+  /**
+   * Return one or more layers the were converted to outlines.
+   */
+  layersByConvertingToOutlines() {
```

Comment:
```suggestion
  toOutlines() {
```

Maybe `toOutlines`, similar to `String.toUppercase` would suffice?

Reply:
Makes sense to me.